### PR TITLE
Improve table styles

### PR DESCRIPTION
**Pull request by julieg18**
# 1/2 `main` <- this <- #2198 
* add vscode prefix to `--checkbox-border` variable
* keep cell spinners and unselected circles the same bg color as cells
* keep the sticky column from losing opacity on row hover 

**Example of cell spinners and unselected circles**

https://user-images.githubusercontent.com/43496356/184926283-3adc63b2-3fa6-4e2f-9771-01fbd682d27e.mov

**Example of now fully opaque experiment column**

https://user-images.githubusercontent.com/43496356/184946778-bc6421e6-ba2d-415e-aa1f-99db7cb738c1.mov

Followup to #2133

### CODE REVIEW

**Review by sroy3 — APPROVED** [review]
Should we change the border to be the same as the row background as well?

**Comment by julieg18** [comment]
> Should we change the border to be the same as the row background as well?

Apologies, I'm not sure what border you're referring to?

**Comment by sroy3** [comment]
> > Should we change the border to be the same as the row background as well?
> 
> Apologies, I'm not sure what border you're referring to?
Screen Shot 2022-08-16 at 11 05 14 AM
 The black border on the spinner where the cursor is.

**Comment by julieg18** [comment]
> The black border on the spinner where the cursor is.

Oh, I see what you're saying! I'll need to rethink this solution a bit more. Now that I'm looking at things a second time, I just noticed that our checkboxes are not visible on the default theme on hover as well: 

image

We may need to rethink our hover background colors again 😅

**Comment by julieg18** [comment]
Properly fixed the background for spinners/bullets and made sure that the exp column always retains full opacity no matter the background color(some row background colors are partially transparent) causing this issue in some themes: 

image

**Review by sroy3 — APPROVED** [review]

**Comment by shcheklein** [comment]
I think things like this should be fixed, folks. Unless I'm missing something and it's an actual edge case (e.g. there are places in VS Code with this theme where it breaks the interface).

Screen Shot 2022-08-16 at 10 37 24 AM

I'm not sure why is it happening btw. @julieg18 could you give a bit more details. What variable do we use for background that causes this? In what case (hover, active, etc?)

**Comment by shcheklein** [comment]
Good, thanks @julieg18 !

**Comment by mattseddon** [comment]
Still a slight issue with the row being selected:

image

File: webview/src/shared/variables.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@ $bg-color: var(--vscode-editor-background);
 $bullet-size: 9px;
 $watermark-color: var(--vscode-descriptionForeground);
 
-$border-color: var(--checkbox-border);
+$border-color: var(--vscode-checkbox-border);
```
Comment (mattseddon) [inline webview/src/shared/variables.scss:6]: There is at least one more of these in this file. `--button-primary-background`

Comment (julieg18) [reply webview/src/shared/variables.scss:6]: Weird, that variable actually doesn't have a `vscode` prefixed version 🤔

**Comment by julieg18** [comment]
> Still a slight issue with the row being selected:

Fixed!

**Pull request merged**